specification/07-theory_operations.adoc

```diff
@@ -288,6 +288,67 @@ must check that:
 The TSM establishes a secured SPDM session with the physical device DSM by going
 through the steps described in the Secured SPDM Session section.
 
+.Device Connection - SPDM Session Establishment
+[source,mermaid]
+....
+%%{init: {'theme': 'neutral', 'themeVariables': {'darkMode': true}, "flowchart" : { "curve" : "basis" } } }%%
+sequenceDiagram
+
+autonumber
+
+participant DSM as Device DSM or ROT
+participant VMM as Host Supervisor Domain Manager (VMM)
+participant TSM
+
+VMM ->> TSM: [COVH] - sbi_covh_connect_device(device_id, stream_id)
+
+Note over TSM,DSM: TSM: SPDM Connection Setup
+
+TSM ->> TSM: Generate SPDM request: GET_VERSION
+TSM ->> VMM: [COVH] - spdm_req(GET_VERSION)
+VMM ->> DSM: [DOE] - SPDM_GET_VERSION
+DSM ->> VMM: [DOE] - SPDM_VERSION
+VMM ->> TSM: [COVH] - spdm_resp(VERSION)
+
+TSM ->> TSM: Generate SPDM request: GET_CAPABILITIES
+TSM ->> VMM: [COVH] - spdm_req(GET_CAPABILITIES)
+VMM ->> DSM: [DOE] - SPDM_GET_CAPABILITIES
+DSM ->> VMM: [DOE] - CAPABILITIES
+VMM ->> TSM: [COVH] - spdm_resp(CAPABILITIES)
+
+TSM ->> TSM: Generate SPDM request: NEGOTIATE_ALGORITHMS
+TSM ->> VMM: [COVH] - spdm_req(NEGOTIATE_ALGORITHMS)
+VMM ->> DSM: [DOE] - SPDM_NEGOTIATE_ALGORITHMS
+DSM ->> VMM: [DOE] - ALGORITHMS
+VMM ->> TSM: [COVH] - spdm_resp(ALGORITHMS)
+
+TSM ->> TSM: Generate SPDM request: GET_CERTIFICATE
+TSM ->> VMM: [COVH] - spdm_req(GET_CERTIFICATE)
+VMM ->> DSM: [DOE] - GET_CERTIFICATE
+DSM ->> VMM: [DOE] - CERTIFICATE
+VMM ->> TSM: [COVH] - spdm_resp(CERTIFICATE)
+TSM ->> TSM: Verify and store device certificate chain
+
+Note over TSM,DSM: TSM: SPDM Key Exchange
+
+TSM ->> TSM: Generate ephemeral SPDM keys K0
+TSM ->> TSM: Generate SPDM request: KEY_EXCHANGE(K0)
+TSM ->> VMM: [COVH] - spdm_req(KEY_EXCHANGE)
+VMM ->> DSM: [DOE] - SPDM_KEY_EXCHANGE
+DSM ->> VMM: [DOE] - KEY_EXCHANGE_RSP
+VMM ->> TSM: [COVH] - spdm_resp(KEY_EXCHANGE_RSP)
+TSM ->> TSM: Derive DHE secret
```

Comment:
Deriver DHE secret, and SPDM session handshake secrete.

Reply:
Done. I added a `Derive SPDM handshake secrets from the DHE secret`step.